### PR TITLE
Boss room readme

### PR DESCRIPTION
**Pull request by Briancoughlin**
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Adding new content for getting started with Boss Room Sample

**Issue Number:**

### CODE REVIEW

File: docs/learn/getting-started-boss-room.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+id: bossroom
+title: Getting Started with Boss Room
+---
+
+![Banner](/img/banner.png)
+# Boss Room - Multiplayer Samples Co-op built with Unity MLAPI
+
+:::important
+This project is currently experimental.
```
Comment (chrispope) [inline docs/learn/getting-started-boss-room.md:10]:
```suggestion
This project is currently in early-access.
```

File: docs/learn/getting-started-boss-room.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+:::
+
+Boss Room is a fully functional Multiplayer Samples Co-op made in Unity and MLAPI. It is built to serve as an educational sample that showcases certain typical gameplay patterns that are frequently featured in similar games.
```
Comment (chrispope) [inline docs/learn/getting-started-boss-room.md:13]:
```suggestion
Boss Room is a fully functional Multiplayer Co-op Sample game made with Unity and MLAPI. We are building the sample to serve as an educational showcase of specific typical gameplay patterns frequently featured in similar games.
```

File: docs/learn/getting-started-boss-room.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+- Make sure to include standalone support for Windows/Mac in your installation. 
+:::
+
+## To add the project with Unity Hub
+1. Open Unity Hub.
+1. Click **Add**.
+1. Select the root folder of the downloaded project.
+
+  ![adding project to unity](/img/addbossroom.gif)
+
```
Comment (chrispope) [inline docs/learn/getting-started-boss-room.md:34]: Should we add a comment saying something like the MLAPI v0.1.0 package will be installed with the Boss Room project?

**Review by chrispope — COMMENTED** [review]
Adding some quick suggestions

**Comment by SamuelBellomo** [comment]
Let's make sure any corrections (like wrong "early access" we make here are also reflected in boss room's readme)

File: docs/learn/getting-started-boss-room.md (hunk 3 above)
Comment (Briancoughlin) [reply docs/learn/getting-started-boss-room.md:34]: I have added this text to the note now

**Comment by lkrell** [comment]
Starting review and verifying the embed code.

File: docs/learn/getting-started-boss-room.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+:::note
+The first time you open the project Unity will import all assets, which will take longer than usual - it is normal.
+The MLAPI v0.1.0 package will be installed with Boss Room.
+
+**Issues with importing due to parental control software**
+
+If you have issues with importing you may want to check your DNS settigns as some ISP parental controls  may block GitHub access.
+
+https://community.virginmedia.com/t5/Networking-and-WiFi/Web-Safe-Breaks-GitHub/td-p/4279652
```
Comment (lkrell) [inline docs/learn/getting-started-boss-room.md:43]:
```suggestion
**Issues with importing due to parental control software**: If you have issues with importing you may want to check your DNS settings as some ISP parental controls  may block GitHub access. For example, see this information on [WebSafe](https://community.virginmedia.com/t5/Networking-and-WiFi/Web-Safe-Breaks-GitHub/td-p/4279652).
```

File: docs/learn/getting-started-boss-room.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+:::
+
+## To open the project for the first time
+
+1. In Unity Hub double click the project.
```
Comment (lkrell) [inline docs/learn/getting-started-boss-room.md:49]:
```suggestion
1. In Unity Hub, open the project.
```

File: docs/learn/getting-started-boss-room.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+1. Navigate to the Project window. 
+1. Open the *Assets* folder
+1. Open the *Scene* Folder
+1. Click the *Startup* scene.
+1. Click  **Play**
```
Comment (lkrell) [inline docs/learn/getting-started-boss-room.md:54]:
```suggestion
1. Open the *Assets* folder.
1. Open the *Scene* folder.
1. Click the *Startup* scene.
1. Click  **Play**.
```

File: docs/learn/getting-started-boss-room.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+
+<iframe src="https://www.youtube.com/embed/OSSBWOJq6gI?playlist=OSSBWOJq6gI&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+
+
+## Testing multiplayer
+
+In order to see the multiplayer functionality in action we can either run multiple instances of the game locally on our computer or choose to connect ver the internet.
```
Comment (lkrell) [inline docs/learn/getting-started-boss-room.md:64]:
```suggestion
In order to see the multiplayer functionality in action, you can either run multiple instances of the game locally on your computer or choose to connect through the internet.
```

File: docs/learn/getting-started-boss-room.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+### Multiplayer over internet
+
+:::tip
+In contrast to running a local setup, when playing over internet we don't neccessarily need a built executable. We can just run the game in editor.
+:::
+
+
+Running the game over internet currently requires setting up a Photon Transport for MLAPI or Alternatively you can use Port Forwarding
```
Comment (lkrell) [inline docs/learn/getting-started-boss-room.md:95]:
```suggestion
Running the game over internet currently requires setting up a [Photon Transport](#photon-transport) for MLAPI or alternatively you can use [Port Forwarding](#port-forwarding).
```

File: docs/learn/getting-started-boss-room.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+#### Photon Transport
+
+Photon Transport uses Photon relay server to facilitate communication between clients and server living on different networks.
+
+:::unity Learn More
+[Boss Room Photon Setup Guide](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/release/v0.1.0-experimental/Documentation/Photon-Realtime/Readme.md)
```
Comment (lkrell) [inline docs/learn/getting-started-boss-room.md:102]:
```suggestion
See the [Boss Room Photon Setup Guide](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/release/v0.1.0-experimental/Documentation/Photon-Realtime/Readme.md) for details.
```

File: docs/learn/getting-started-boss-room.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
+---
+id: bossroom
+title: Getting Started with Boss Room
+---
+
+![Banner](/img/banner.png)
+# Boss Room - Multiplayer Samples Co-op built with Unity MLAPI
```
Comment (lkrell) [inline docs/learn/getting-started-boss-room.md:7]: I would remove this

**Review by lkrell — COMMENTED** [review]
edits for ya

**Review by lkrell — APPROVED** [review]
Approving once these are done, good to go

File: docs/learn/getting-started-boss-room.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+<iframe src="https://www.youtube.com/embed/OSSBWOJq6gI?playlist=OSSBWOJq6gI&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+
+
+## Testing multiplayer
+
+In order to see the multiplayer functionality in action we can either run multiple instances of the game locally on our computer or choose to connect ver the internet.
+
+---------------
```
[the remark this answers is not in the document]
Comment (Briancoughlin) [reply docs/learn/getting-started-boss-room.md]: line removed

File: docs/learn/getting-started-boss-room.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+
+**Issues with importing due to parental control software**
+
+If you have issues with importing you may want to check your DNS settigns as some ISP parental controls  may block GitHub access.
+
+https://community.virginmedia.com/t5/Networking-and-WiFi/Web-Safe-Breaks-GitHub/td-p/4279652
+
+:::
+
+## To open the project for the first time
```
[the remark this answers is not in the document]
Comment (Briancoughlin) [reply docs/learn/getting-started-boss-room.md:52]: section updated

File: docs/learn/getting-started-boss-room.md (hunk 7 above)
Comment (Briancoughlin) [reply docs/learn/getting-started-boss-room.md]: updated

**Pull request merged**